windows/installer-source/KaliteSetupScript.iss

```diff
@@ -337,6 +337,25 @@ begin
     end;
 end;
 
+procedure HandlePipSetup;
+var
+    PipCommand: string;
+    PipPath: string;
+    ErrorCode: integer;
+
+begin
+    PipPath := 'C:\Python27\Scripts\pip.exe';
+    PipCommand := 'install ' + '"' + ExpandConstant('{app}') + '\ka-lite\dist\ka-lite-static-0.14.dev10.zip' + '"';

```

Comment:
Hardcoded? We probably want to make it more general. Maybe a shell glob? `ka-lite-static-*.zip`?